actionners/kubernetes/labelize/labelize.go

```diff
@@ -23,19 +24,52 @@ type patch struct {
 
 const (
 	metadataLabels = "/metadata/labels/"
+	podStr         = "pod"
+	nodeStr        = "node"
 )
 
 func Action(action *rules.Action, event *events.Event) (utils.LogLine, error) {
-	pod := event.GetPodName()
+	podName := event.GetPodName()
 	namespace := event.GetNamespaceName()
 
-	objects := map[string]string{
-		"pod":       pod,
-		"namespace": namespace,
-	}
+	objects := map[string]string{}
 
 	payload := make([]patch, 0)
 	parameters := action.GetParameters()
+
+	client := kubernetes.GetClient()
+
+	var err error
+	var kind string
+	var node *corev1.Node
+
+	if parameters["level"] == nodeStr {
+		kind = parameters["level"].(string)
```

Comment:
Will the pod be required in the event when using a rule for labeling a node? I mean my question only makes sense if Falco could generate events from node actions too, does it?

Reply:
yep it's required, to know find the node where the pod is running. sadly, for the events coming from the host, i dug into the rules, most of them don't provide enough details to know which host is concerned for now